test/index.js

```diff
@@ -547,6 +547,30 @@ describe('Nightmare', function () {
       value.should.equal('');
     })
 
+    it('should not type in a nonexistent selector', function*(){
+      var failed = false;
+      try {
+        yield nightmare
+          .goto(fixture('manipulation'))
+          .type('does-not-exist', 'nightmare');
+      } catch(e){
+        failed = true;
+      }
+      failed.should.be.true;
```

Comment:
@Mr0grog totally fine! :)

Frankly, I'm not sure that `mocha-generators` should be included considering the "official line" is to use promises instead.  That feels like a topic for conversation elsewhere, though, and may very well be my intentions getting away from me.

That said, `mocha-generators` should (ha) probably patch `throw`.  I'll throw (double ha) that on my list of things to look at.  (Of course, this assumes you haven't done it already.  I haven't set up the ol' @Mr0grog filter on `mocha-generators` yet. ;) )